### ISSUE TITLE
Filter out records outside Australia by default

### ISSUE THREAD

**Issue by javier-molina** [AtlasOfLivingAustralia/biocache-hubs#582]
As a user searching for occurrence records, I want to be shown only records in Australia by default.

**Background**
ALA occurrence record database contains records from across the globe despite the majority being from Australia. The current display is not a heat-map and is difficult to ascertain where the volume of records is.

![Screenshot 2023-08-25 at 2 16 06 pm](https://github.com/AtlasOfLivingAustralia/biocache-hubs/assets/8326394/1c50af13-d958-46c1-8bee-132158a00dc9)

It is however most likely than a user coming to the ALA will be more interested in local records hence displaying only records in Australia is a sensible default.

![Screenshot 2023-08-25 at 2 03 54 pm](https://github.com/AtlasOfLivingAustralia/biocache-hubs/assets/8326394/7d072f6f-a0e6-4f29-a8c2-e1ea9e00e774)

**Requirements**

- Add an additional filter to the ALA General profile to include only records in Australia.
- As with fine-grained filters in the data profiles, it should be possible to remove this particular filter.

 
**Question**
- What is the best spatial layer that would represent Australian territories and waters?

**Comment by adam-collins** [comment]
I do not think we should make this decision for every user of the biocache. I think it would be better if this was addressed on the simple search page https://biocache.ala.org.au/search#tab_simpleSearch. e.g. adding a button along the lines of 'Everything in Australia', 'Australian Data', etc. Delivering these results though a data quality filter is appropriate.

I expect that for the user in this use case a combination of spatial, species and data resource filters will be required in addition to existing data quality filter items. An expanded profile of this user is required to identify their specific areas of interest. 
- Terrestrial only?
- Coastal areas?
- Territories?
- A rectangle around Australia?
- Native species only?
- A subset of data resources?
- Citizen science records?
- Any relevant data quality profiles? Do they need further targeting for this user?
